file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237
        throw new Error(y, { cause: u });
              ^

Error: [11-Apr-2025 22:26:58 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>DELETE FROM wp_securio_bf_bans WHERE date_added < 1743200818</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: DELETE FROM `wp_securio_bf_bans` WHERE `date_added` &lt; 1743200818 | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 1 no such table: wp_securio_bf_bans
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(690): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 0, Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(472): WP_SQLite_Driver->query('DELETE FROM wp_...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(413): WP_SQLite_DB->_do_query('DELETE FROM wp_...')
#3 /wordpress/wp-content/plugins/securio-wplogin/securio-wplogin.php(182): WP_SQLite_DB->query('DELETE FROM wp_...')
#4 /wordpress/wp-content/plugins/securio-wplogin/securio-wplogin.php(165): Securio_WPLogin->cleanup()
#5 /wordpress/wp-content/plugins/securio-wplogin/securio-wplogin.php(76): Securio_WPLogin->__construct()
#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('securio-wplogin...')
#8 /internal/eval.php(20): activate_plugin('securio-wplogin...')
#9 {main}</pre>
 for query DELETE FROM wp_securio_bf_bans WHERE date_added < 1743200818 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/securio-wplogin/securio-wplogin.php'), Securio_WPLogin->__construct, Securio_WPLogin->cleanup, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[11-Apr-2025 22:26:58 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>DELETE FROM wp_securio_bf_fails WHERE date_added < 1743200818</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: DELETE FROM `wp_securio_bf_fails` WHERE `date_added` &lt; 1743200818 | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 1 no such table: wp_securio_bf_fails
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(690): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 0, Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(472): WP_SQLite_Driver->query('DELETE FROM wp_...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(413): WP_SQLite_DB->_do_query('DELETE FROM wp_...')
#3 /wordpress/wp-content/plugins/securio-wplogin/securio-wplogin.php(186): WP_SQLite_DB->query('DELETE FROM wp_...')
#4 /wordpress/wp-content/plugins/securio-wplogin/securio-wplogin.php(165): Securio_WPLogin->cleanup()
#5 /wordpress/wp-content/plugins/securio-wplogin/securio-wplogin.php(76): Securio_WPLogin->__construct()
#6 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#7 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('securio-wplogin...')
#8 /internal/eval.php(20): activate_plugin('securio-wplogin...')
#9 {main}</pre>
 for query DELETE FROM wp_securio_bf_fails WHERE date_added < 1743200818 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/securio-wplogin/securio-wplogin.php'), Securio_WPLogin->__construct, Securio_WPLogin->cleanup, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[11-Apr-2025 22:27:07 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>DELETE FROM wp_securio_bf_bans WHERE date_added < 1743200827</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: DELETE FROM wp_securio_bf_bans WHERE date_added &lt; 1743200827 | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_securio_bf_bans.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/securio-wplogin/securio-wplogin.php(182): WP_SQLite_DB->query('DELETE FROM wp_...')
#2 /wordpress/wp-content/plugins/securio-wplogin/securio-wplogin.php(165): Securio_WPLogin->cleanup()
#3 /wordpress/wp-content/plugins/securio-wplogin/securio-wplogin.php(76): Securio_WPLogin->__construct()
#4 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#5 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('securio-wplogin...')
#6 /internal/eval.php(20): activate_plugin('securio-wplogin...')
#7 {main}</pre>
 for query DELETE FROM wp_securio_bf_bans WHERE date_added < 1743200827 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/securio-wplogin/securio-wplogin.php'), Securio_WPLogin->__construct, Securio_WPLogin->cleanup, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[11-Apr-2025 22:27:07 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>DELETE FROM wp_securio_bf_fails WHERE date_added < 1743200827</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: DELETE FROM wp_securio_bf_fails WHERE date_added &lt; 1743200827 | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_securio_bf_fails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/securio-wplogin/securio-wplogin.php(186): WP_SQLite_DB->query('DELETE FROM wp_...')
#2 /wordpress/wp-content/plugins/securio-wplogin/securio-wplogin.php(165): Securio_WPLogin->cleanup()
#3 /wordpress/wp-content/plugins/securio-wplogin/securio-wplogin.php(76): Securio_WPLogin->__construct()
#4 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#5 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('securio-wplogin...')
#6 /internal/eval.php(20): activate_plugin('securio-wplogin...')
#7 {main}</pre>
 for query DELETE FROM wp_securio_bf_fails WHERE date_added < 1743200827 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/securio-wplogin/securio-wplogin.php'), Securio_WPLogin->__construct, Securio_WPLogin->cleanup, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[11-Apr-2025 22:27:16 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>DELETE FROM wp_securio_bf_bans WHERE date_added < 1743200836</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: DELETE FROM wp_securio_bf_bans WHERE date_added &lt; 1743200836 | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_securio_bf_bans.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/securio-wplogin/securio-wplogin.php(182): WP_SQLite_DB->query('DELETE FROM wp_...')
#2 /wordpress/wp-content/plugins/securio-wplogin/securio-wplogin.php(165): Securio_WPLogin->cleanup()
#3 /wordpress/wp-content/plugins/securio-wplogin/securio-wplogin.php(76): Securio_WPLogin->__construct()
#4 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#5 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('securio-wplogin...')
#6 /internal/eval.php(20): activate_plugin('securio-wplogin...')
#7 {main}</pre>
 for query DELETE FROM wp_securio_bf_bans WHERE date_added < 1743200836 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/securio-wplogin/securio-wplogin.php'), Securio_WPLogin->__construct, Securio_WPLogin->cleanup, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[11-Apr-2025 22:27:16 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>DELETE FROM wp_securio_bf_fails WHERE date_added < 1743200836</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: DELETE FROM wp_securio_bf_fails WHERE date_added &lt; 1743200836 | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_securio_bf_fails.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/securio-wplogin/securio-wplogin.php(186): WP_SQLite_DB->query('DELETE FROM wp_...')
#2 /wordpress/wp-content/plugins/securio-wplogin/securio-wplogin.php(165): Securio_WPLogin->cleanup()
#3 /wordpress/wp-content/plugins/securio-wplogin/securio-wplogin.php(76): Securio_WPLogin->__construct()
#4 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#5 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('securio-wplogin...')
#6 /internal/eval.php(20): activate_plugin('securio-wplogin...')
#7 {main}</pre>
 for query DELETE FROM wp_securio_bf_fails WHERE date_added < 1743200836 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/securio-wplogin/securio-wplogin.php'), Securio_WPLogin->__construct, Securio_WPLogin->cleanup, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[11-Apr-2025 22:27:27 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method serverDown::init() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324
Stack trace:
#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#2 /wordpress/wp-settings.php(704): do_action('init')
#3 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#4 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#5 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#6 {main}
  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324
[11-Apr-2025 22:27:35 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method serverDown::init() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324
Stack trace:
#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#2 /wordpress/wp-settings.php(704): do_action('init')
#3 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#4 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#5 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#6 {main}
  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324
[11-Apr-2025 22:27:44 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method serverDown::init() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324
Stack trace:
#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#2 /wordpress/wp-settings.php(704): do_action('init')
#3 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#4 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#5 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#6 {main}
  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324
[11-Apr-2025 22:29:51 UTC] PHP Deprecated:  Required parameter $resource_type follows optional parameter $globals in /wordpress/wp-content/plugins/sj-http2-server-push-optimization/classes/class-sjh-model.php on line 77
[11-Apr-2025 22:29:51 UTC] PHP Deprecated:  Required parameter $resource_type follows optional parameter $globals in /wordpress/wp-content/plugins/sj-http2-server-push-optimization/classes/class-sjh-model.php on line 77
[11-Apr-2025 22:29:59 UTC] PHP Deprecated:  Required parameter $resource_type follows optional parameter $globals in /wordpress/wp-content/plugins/sj-http2-server-push-optimization/classes/class-sjh-model.php on line 77
[11-Apr-2025 22:30:00 UTC] PHP Deprecated:  Required parameter $resource_type follows optional parameter $globals in /wordpress/wp-content/plugins/sj-http2-server-push-optimization/classes/class-sjh-model.php on line 77
[11-Apr-2025 22:30:08 UTC] PHP Deprecated:  Required parameter $resource_type follows optional parameter $globals in /wordpress/wp-content/plugins/sj-http2-server-push-optimization/classes/class-sjh-model.php on line 77
[11-Apr-2025 22:30:09 UTC] PHP Deprecated:  Required parameter $resource_type follows optional parameter $globals in /wordpress/wp-content/plugins/sj-http2-server-push-optimization/classes/class-sjh-model.php on line 77

    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
  [cause]: Error: Error when executing the blueprint step #5 ({"step":"installPlugin","pluginData":{"resource":"wordpress.org/plugins","slug":"slug-smart-replacer"},"options":{"activate":true}}) : Could not download "https://downloads.wordpress.org/plugin/slug-smart-replacer.latest-stable.zip".
  				Check if the URL is correct and the server is reachable.
  				If it is reachable, the server might be blocking the request.
  				Check the browser console and network tabs for more information.
  
  				## Does the console show the error "No 'Access-Control-Allow-Origin' header"?
  
  				This means the server that hosts your file does not allow requests from other sites
  				(cross-origin requests, or CORS).	You need to move the asset to a server that allows
  				cross-origin file downloads. Learn more about CORS at
  				https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.
  
  				If your file is on GitHub, load it from "raw.githubusercontent.com".
  				Here's how to do that:
  
  				1. Start with the original GitHub URL of the file. For example:
  				https://github.com/username/repository/blob/branch/filename.
  				2. Replace "github.com" with "raw.githubusercontent.com".
  				3. Remove the "/blob/" part of the URL.
  
  				The resulting URL should look like this:
  				https://raw.githubusercontent.com/username/repository/branch/filename
  
  				Error:
  				Error: Could not download "https://downloads.wordpress.org/plugin/slug-smart-replacer.latest-stable.zip"
      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)
      at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)
      at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
    [cause]: Error: Could not download "https://downloads.wordpress.org/plugin/slug-smart-replacer.latest-stable.zip".
    				Check if the URL is correct and the server is reachable.
    				If it is reachable, the server might be blocking the request.
    				Check the browser console and network tabs for more information.
    
    				## Does the console show the error "No 'Access-Control-Allow-Origin' header"?
    
    				This means the server that hosts your file does not allow requests from other sites
    				(cross-origin requests, or CORS).	You need to move the asset to a server that allows
    				cross-origin file downloads. Learn more about CORS at
    				https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.
    
    				If your file is on GitHub, load it from "raw.githubusercontent.com".
    				Here's how to do that:
    
    				1. Start with the original GitHub URL of the file. For example:
    				https://github.com/username/repository/blob/branch/filename.
    				2. Replace "github.com" with "raw.githubusercontent.com".
    				3. Remove the "/blob/" part of the URL.
    
    				The resulting URL should look like this:
    				https://raw.githubusercontent.com/username/repository/branch/filename
    
    				Error:
    				Error: Could not download "https://downloads.wordpress.org/plugin/slug-smart-replacer.latest-stable.zip"
        at Ct.resolve (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:1472:13)
        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
        at async _.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/util/index.js:45:14)
  }
}

Node.js v23.11.0
